templates/target.jade

```diff
@@ -0,0 +1,13 @@
+doctype html
+html
+	head
+		title= title ? title + ' – BookBrainz' : 'BookBrainz – The Open Book Database'
+		link(rel='stylesheet', href='/stylesheets/style.css')
+		link(rel='stylesheet', href='/stylesheets/select2.min.css')
+		link(rel='stylesheet', href='/stylesheets/select2-bootstrap.min.css')
+
+		include favicon
+	body
```

Comment:
You'll want to put the props in here too somewhere, so that they can be accessed by the browser when the React components are being refreshed on the client side.